sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/BeamSqlDslUdfUdafTest.java

```diff
@@ -91,6 +84,23 @@ public void testJodaTimeUdfUdaf() throws Exception {
     pipeline.run().waitUntilFinish();
   }
 
+  @Ignore("Fix the access to DateTime field of Row in Udaf")
```

Comment:
Current change breaks user defined aggregation function. Need to fix it before merging PR.